### PR TITLE
Add AzureML Quickstart Guides

### PR DESCRIPTION
**Pull request by ouphi**
# Add AzureML quickstarts for YOLOv5 and YOLOv8

Added 2 quickstarts:
- YOLOv5 with AzureML
- YOLOv8 with AzureML

The related issue is https://github.com/ultralytics/ultralytics/issues/4548

### Summary
🚀📝🌐

This pull request adds two new markdown files to the `docs` folder that provide quickstart guides for using YOLOv8 and YOLOv5 on AzureML. The files explain how to create a compute instance, run the models from a terminal or a notebook, and provide code snippets, images, and links. The purpose is to make it easy for users to experiment with the models on a cloud platform.

> _`YOLOv8` preview_
> _AzureML quickstart guides_
> _Winter of learning_

### Walkthrough
*  Add two quickstart guides for using YOLOv8 and YOLOv5 on AzureML ([link](https://github.com/ultralytics/ultralytics/pull/4772/files?diff=unified&w=0#diff-4d9df1e63fa723a03608a47fc58ecd4695062fa379461a3f12dfdba1d1fc07e8R1-R119), [link](https://github.com/ultralytics/ultralytics/pull/4772/files?diff=unified&w=0#diff-1fb99daacc1774c37bea418ee45d2fff72f01ed20d4e66f7298113019e0a615eR1-R96))

## 🛠️ PR Summary

<sub>Made with ❤️ by [Ultralytics Actions](https://github.com/ultralytics/actions)<sub>

### 🌟 Summary
New AzureML quickstart guides for YOLOv5 and YOLOv8.

### 📊 Key Changes
- Added Azure Machine Learning quickstart guides for YOLOv5 and YOLOv8 within the documentation.
- Created a new section for AzureML in the YOLOv5 environment documentation.
- Introduced setup and usage instructions, including virtual environments and kernel configuration.
- Provided examples of how to train, validate, and run predictions using YOLO models on AzureML.
- Updated the documentation index to include AzureML Quickstart.

### 🎯 Purpose & Impact
- Enables users to quickly get started with YOLO models on AzureML, promoting easier experimentation and adoption.
- Provides clear instructions for setting up the environment and running key tasks, enhancing the user experience for both researchers and practitioners.
- Helps expand the accessibility of Ultralytics' machine learning tools to Azure's cloud computing services, potentially broadening the user base.

### CODE REVIEW

File: docs/yolov5/index.md [hunk 1]
```diff
@@ -57,6 +57,7 @@ YOLOv5 is designed to be run in the following up-to-date, verified environments,
   GPU: <a href="https://bit.ly/yolov5-paperspace-notebook"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"></a> <a href="https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/ultralytics/yolov5"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
 - **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](environments/google_cloud_quickstart_tutorial.md)
 - **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](environments/aws_quickstart_tutorial.md)
+- **AzureML** Worksaces and compute instances. See [AzureML Quickstart Guide](environments/azureml_quickstart_tutorial.md)
```
Comment (ouphi) [inline docs/yolov5/index.md:60]:
```suggestion
- **Azure** Azure Machine Learning. See [AzureML Quickstart Guide](environments/azureml_quickstart_tutorial.md)
```

**Comment by glenn-jocher** [comment]
@ouphi thanks for the fix! I think this PR is all set now :)

**Comment by ouphi** [comment]
@glenn-jocher Yes it is all set :) Thanks for polishing the PR and adding all the links!

**Comment by glenn-jocher** [comment]
@ouphi PR merged! Thank you for your contributions to YOLO :)

**Pull request merged**